legal-api/report-templates/template-parts/continuation/effectiveDate.html

```diff
@@ -3,7 +3,7 @@
     <div class="section-title mt-4">Continuation Effective Date</div>
     <div class="section-data mt-4">
         {% if header.isFutureEffective %}
-            {{effective_date_time}}
+            The continuation is to take effect on {{effective_date_time}}.
```

Comment:
Do you have a sample output?

(Is effective_date_time something like, "October 7, 2024 at 8:00 pm Pacific time"?)

Reply:
![image](https://github.com/user-attachments/assets/b39d01a0-579d-44fc-b0d8-38e872d8d89a)

Reply:
Excellent, thanks. Resolving.